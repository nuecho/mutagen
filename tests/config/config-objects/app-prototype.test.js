cfgObjectTests("appPrototypes", {initialNumberOfObjects: 6});