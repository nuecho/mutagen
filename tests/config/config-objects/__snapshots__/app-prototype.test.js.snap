// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen appPrototypes config import-export] should create the appPrototypes: the appPrototypes have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ appprototype => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen appPrototypes config import-export] should throw exception when new appPrototypes to import miss mandatory properties appPrototypes: exception has been thrown 1`] = `
Object {
  "stderr": "Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
	Mandatory properties [type, version] not set in appprototype 'cfgObjectTest'
",
  "stdout": "",
}
`;

exports[`[mutagen appPrototypes config import-export] should update the modified appPrototypes: the modified appPrototypes have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ appprototype => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen appPrototypes config import-export] should update the same appPrototypes: the appPrototypes have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ appprototype => cfgObjectTest
Completed. 1 object(s) imported.
",
}
`;
